vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"k8s.io/autoscaler/vertical-pod-autoscaler/common"
 	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/input/history"
+	_ "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/model"
```

Comment:
I think you can remove this import

Reply:
With the [above](https://github.com/kubernetes/autoscaler/pull/2905#discussion_r390289950) change, yes.

Reply:
Now, I can't anymore because of the need to [initialise](https://github.com/kubernetes/autoscaler/pull/2905/files#diff-483c01ba6f34fd6f8c9d4944684cff98R67) aggregations config struct singleton instance.